content/docs/fedora-releases-guide.md

```diff
@@ -0,0 +1,138 @@
+---
+title: How to do Fedora releases with Packit
+weight: 9
+---
+# How to do Fedora releases with Packit
+
+Let's split the release process into single steps:
+1. New upstream release 
+2. Uploading archive to lookaside cache
+3. Updating dist-git content 
+4. Koji builds
+5. Bodhi updates
+
+Doing Fedora releases with Packit means utilising our 3 jobs - `propose_downstream`, `koji_build` and 
+`bodhi_update` - where each of the jobs takes care of a different part of the release process.
+
+## Propose downstream job
+For enabling the propose downstream job, you need to have [Packit Service installed](/docs/packit-service) 
+and have a `propose_downstream` job in the configuration file for the given upstream repository.
+The [propose_downstream job](/docs/configuration/#propose_downstream) should be then configured like this:
+
+```yaml
+jobs:
+- job: propose_downstream
+  trigger: release
+  dist_git_branches:
+    - main
+```
+You can adjust the `dist_git_branches` field to include the
+dist-git branches you want to update and also utilise [aliases](/docs/configuration/#aliases) 
+instead of using hardcoded versions.
+
+#### New upstream release
+The process of releasing a new version starts in the upstream repository by creating a 
+new upstream release. Packit gets the information about the newly created release from GitHub,
+loads the config from the release commit and if there is a `propose_downstream` job
+defined, the workflow begins.
+Users with write or admin permissions to the repository can retrigger an
+update via a comment in any open issue:
+
+    /packit propose-downstream
+
+#### Uploading archive to lookaside cache
+The upstream archive needs to be downloaded by Packit at first and then uploaded to the lookaside cache.
+By default, Packit downloads sources defined in the specfile that contain URLs.
+You can override these URLs via [`sources`](/docs/configuration#sources) configuration key.
+
+For Python packages, you can use a 
+[GitHub action](https://packaging.python.org/guides/publishing-package-distribution-releases-using-github-actions-ci-cd-workflows)
+([example setup of Packit itself](https://github.com/packit/packit/blob/main/.github/workflows/pypi-publish.yml))
+that automatically builds and uploads the archive to PyPI 
+on each new release. Then during `propose_downstream`, Packit tries to download the archive from the PyPI and 
+also utilises retries in case the upstream archive is not 
+available in the time of running the job.
+
+#### Updating dist-git content
+After saving the archive in the lookaside cache,
+Packit updates the dist-git content (mainly `sources` file and spec-file) via pull requests for the specified branches.
```

Comment:
I'd suggest using `spec file` instead of `spec-file`. We discussed the same thing in https://github.com/packit/specfile and ended up going with the latter. However, this may be a problem in more places in the docs.